dll/kernel32.cpp

```diff
@@ -1823,6 +1828,19 @@ namespace kernel32 {
 		}
 	}
 
+	int WIN_FUNC GetLocaleInfoW(unsigned int Locale, int LCType, wchar_t *lpLCData, int cchData) {
+		DEBUG_LOG("GetLocaleInfoW %d %d\n", Locale, LCType);
+		std::string info = str_for_LCType(LCType);
+		std::wstring ret(ret.begin(), ret.end());
+
+		if (!cchData) {
+			return ret.size() + 1;
+		} else {
+			memcpy(lpLCData, ret.c_str(), ret.size() + sizeof(wchar_t));
```

Comment:
This will copy N+2 bytes, not 2N+2

(Would also be nice to assert that the size is <= cchData, here and in GetLocaleInfoA, so we get a safe crash instead of UB)